lsp-mode.el

```diff
@@ -650,6 +650,26 @@ If this is set to nil, `eldoc' will show only the symbol information."
   :type 'boolean
   :group 'lsp-mode)
 
+(defcustom lsp-modeline-code-actions-enable nil
+  "Wheter to show code actions on modeline."
+  :type 'boolean
+  :group 'lsp-mode)
+
+(defcustom lsp-modeline-code-actions-delay 0.2
```

Comment:
we should never create timers but use `lsp-on-*` hooks.

Reply:
Yeah, what hook do you think I should use here?

Reply:
Changed to `lsp-on-idle-hook`